index.html

```diff
@@ -173,5 +239,56 @@ <h4 style="font-weight: lighter;">We know how large objects will act, but things
             </div>
             </div>
         </section>
+
+        <section class="why-us">
+        <div class="why-choose-us">
+            <h2>WHY CHOOSE US</h2>
```

Comment:
I don't see h2 and the paragraph below in your css. I think you need to give classes to them and style them. They don't look like in Figma. They are supposed to have different weight, line-height, etc. Check it out :)